src/main/java/com/alysaa/geyserupdater/common/util/ScriptCreator.java

```diff
@@ -54,6 +52,10 @@ public static void createScript(boolean runLoop) throws IOException {
                     dos.writeBytes("echo \"Server stopped, restarting in 10 seconds!\"; sleep 10; done\n");
                 }
             }
+            System.out.println("[GeyserUpdater] A custom restart script has been made for you.");
+            if (runLoop) {
+                System.out.println("[GeyserUpdater] You will need to shutdown the server and use our provided restart script.");
+            }
```

Comment:
Not really happy with this, but shutting down the server is only really applicable for BungeeCord. Spigot users can use the /restart command immediately, and don't need to use the generated script to start the server.  

Any ideas?